src/Inventory.zig

```diff
@@ -1554,6 +1555,8 @@ pub const Command = struct { // MARK: Command
 	const UpdateBlock = struct { // MARK: UpdateBlock
 		source: InventoryAndSlot,
 		pos: Vec3i,
+		dropPos: Vec3d,
```

Comment:
In my opinion it would make more sense to use a relative position (relative to `pos`), after all that's what you got on the client side anyways.